conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
If anyone is interested it is currently available and under development at https://github.com/tudelft/tudelft_vision.